client/src/graphql_utils/graphql_utils.js

```diff
@@ -75,9 +75,8 @@ let client = null;
 export function get_client() {
   if (!client) {
     client = new ApolloClient({
-      link: new BatchHttpLink({
+      link: createHttpLink({
```

Comment:
Haven't switched back to `compressToBase64`, let me know if that's necessary here